test/distributed/_composable/test_compose.py

```diff
@@ -76,7 +76,9 @@ def test_wrap_same_submodule(self, use_reentrant: bool):
 
         test_model = copy.deepcopy(model)
         # compose checkpoint and fully_shard
-        test_model.seq = checkpoint(test_model.seq, use_reentrant=use_reentrant)
+        test_model.seq = checkpoint(
+            test_model.seq, use_reentrant=use_reentrant
```

Comment:
nit: let's try to avoid formatting changes and logical changes in the same PR as it makes it harder to review.